dev-client/src/translations/en.json

```diff
@@ -294,9 +291,9 @@
     "example_email": "name@domain.com",
     "profile_image_alt": "Profile image",
     "role": {
-      "viewer": "Viewer",
-      "contributor": "Contributor",
-      "manager": "Manager"
+      "MANAGER": "Manager",
+      "CONTRIBUTOR": "Contributor",
+      "VIEWER": "Viewer"
```

Comment:
question: why are these all-caps? why did the order change?

Reply:
because they are constant enum values (i.e. they are actually values that get passed around in the app, not just i18n keys), so they're delineated by being caps lock. in practice, it means you can't change this key by just changing it and `CTRL+F`ing for `MANAGER` because it might be getting passed in from a variable, so the caps are saying "be careful when changing this!"